packages/ketchup/src/f-components/f-image/f-image.tsx

```diff
@@ -59,6 +59,7 @@ export const FImage: FunctionalComponent<FImageProps> = (
             style={style}
             title={props.title}
             onClick={props.onClick}
+            tabindex="0"
```

Comment:
this will make every fimage in the library to be focusable by pressing tab, why the need to do this?

Reply:
added a new property in f-image props to specify tab index if it needs to be used